Packs/GraphQL/Integrations/GraphQL/GraphQL.yml

```
@@ -17,6 +17,7 @@ configuration:
   defaultvalue: 'true'
```

```suggestion
  defaultvalue: 'false'
```
Also, update the description to note that it is not supported anymore.